tasks/components/layout/TaskTemplateDisplay.tsx

```diff
@@ -0,0 +1,94 @@
+import { tw } from '@helpwave/common/twind'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { LucideArrowLeftRight } from 'lucide-react'
+import { TaskTemplateCard } from '../cards/TaskTemplateCard'
+import { Button } from '@helpwave/common/components/Button'
+import { AddCard } from '../cards/AddCard'
+import { useRouter } from 'next/router'
+import type { TaskTemplateDTO } from '../../mutations/task_template_mutations'
+import { Span } from '@helpwave/common/components/Span'
+
+export type TaskTemplateDisplayTranslation = {
+  addNewTaskTemplate: string,
+  switchToPersonal: string,
+  switchToWardView: string,
+  taskTemplates: string
+}
+
+const defaultTaskTemplateDisplayTranslation = {
+  en: {
+    addNewTaskTemplate: 'Add new template',
+    switchToPersonal: 'Personal Task Templates',
+    switchToWardView: 'Ward Task Templates',
+    taskTemplates: 'Task Templates'
```

Comment:
We should reword this to: `Ward Task Templates`  or `Personal Task Templates`

Reply:
That's exactly what stands there, or did you mean the German translation too

Reply:
![image](https://user-images.githubusercontent.com/13922835/236341447-e978a9c2-93ed-42e6-8fc6-9c01c95b6a2a.png)